tempodb/backend/raw.go

```diff
@@ -129,7 +131,17 @@ func (r *reader) ReadRange(ctx context.Context, name string, blockID uuid.UUID,
 }
 
 func (r *reader) Tenants(ctx context.Context) ([]string, error) {
-	return r.r.List(ctx, nil)
+	list, err := r.r.List(ctx, nil)
+
+	// this filter is added to fix a GCS usage stats issue that would result in ""
+	filteredList := make([]string, 0)
```

Comment:
nit: `make(..., 0)` isn't common.  We can alloc with the capacity = `len(list)` or if going with empty, then prefer `var filteredList []string`